### PR TITLE
Port to sourced-ml 0.8

### PR DESCRIPTION
**Pull request by warenlg**
Signed-off-by: Waren Long <waren@sourced.tech>

### CODE REVIEW

**Comment by zurk** [comment]
@vmarkovtsev I see that FormatModel missing `DESCRIPTION` and `LICENSE` fields. 
Should we add them?
I can take care of it as soon as Waren on vacations.

**Comment by warenlg** [comment]
Thanks for noticing @zurk I just amended my commit to add it

File: lookout/style/format/model.py [hunk 1]
```diff
@@ -26,6 +27,8 @@ class FormatModel(AnalyzerModel):
 
     NAME = "code-format"
     VENDOR = "source{d}"
+    DESCRIPTION = "Model that contains Rules to format code style."
```
Comment (vmarkovtsev) [inline lookout/style/format/model.py]:
```suggestion
    DESCRIPTION = "Model that contains Rules to format code."
```

File: setup.py [hunk 2]
```diff
@@ -33,7 +33,7 @@
     namespace_packages=["lookout"],
     keywords=["machine learning on source code", "babelfish", "lookout"],
     install_requires=[
-        "sourced-ml>=0.7.2,<0.8",
+        "sourced-ml>=0.8.0,<0.9",
         "lookout-sdk-ml>=0.9.0,<0.10",
```
Comment (vmarkovtsev) [inline setup.py]: sdk-ml needs to be updated to 0.11

Comment (warenlg) [reply setup.py]: The purpose of this PR was to alleviate #592 with some test fixes.
Updating to sdk-ml 0.11 Requires a lot of work Thzt is done in the other PR

**Comment by vmarkovtsev** [comment]
OK, @zurk please finish this one

**Comment by warenlg** [comment]
To pass the tests, this PR requires https://github.com/src-d/modelforge/pull/79 to be merged

**Comment by vmarkovtsev** [comment]
Thank you @warenlg @zurk this one was tough.

**Pull request merged**